cli/command/load_balancer_command.go

```diff
@@ -376,6 +388,37 @@ func (l *LoadBalancerCommand) HealthCheck(c *cli.Context) error {
 	return l.Printer.PrintLoadBalancerHealthCheck(loadBalancer)
 }
 
+func (l *LoadBalancerCommand) IdleTimeout(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "NAME", "TIMEOUT")
+	if err != nil {
+		return err
+	}
+
+	id, err := l.resolveSingleID("load_balancer", args["NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.Client.GetLoadBalancer(id)
+	if err != nil {
+		return err
+	}
+
+	if timeout := args["TIMEOUT"]; timeout != "" {
```

Comment:
this will never happen because `extractArgs` would error if it was not specified